client/components/domains/domain-product-price/index.jsx

```diff
@@ -152,19 +155,28 @@ class DomainProductPrice extends Component {
 						components: { small: <small /> },
 					} ) }
 				</div>
-				<div className="domain-product-price__renewal-price">
+				<div className="domain-product-price__regular-price">
 					{ translate( '%(cost)s {{small}}/year{{/small}}', {
 						args: { cost: price },
 						components: { small: <small /> },
 						comment: '%(cost)s is the annual renewal price of a domain currently on sale',
 					} ) }
 				</div>
+				{ isRenewCostDifferent && (
+					<div className="domain-product-price__renewal-price">
+						{ translate( 'Renews for %(cost)s {{small}}/year{{/small}}', {
+							args: { cost: renewPrice },
+							components: { small: <small /> },
+							comment: '%(cost)s is the annual renewal price of the domain',
+						} ) }
+					</div>
+				) }
```

Comment:
What do you think of wrapping this code in a render function, to avoid duplication code?